### DISCUSSION TITLE
[enhancement] Add hash images blacklist

### DISCUSSION THREAD

**Discussion by TuskarMA** [umputun/tg-spam#154] [Ideas]
Some spammers only post images, while our Telegram chat has specific topics for users to share images. Why not implement a simple check that computes the MD5 hash of each image and compares it against an existing blacklist?

**Comment by umputun** [comment]
It doesn't sound too hard to implement; however, I'm not aware of "an existing blacklist." Do you have some examples of those lists publicly available?

**Comment by TuskarMA** [comment]
We use our own list for moderation in Discord. Unfortunately, I cannot provide the list of hashes, and I don't think it would be useful for Telegram either, due to the specific nature of spam images we handle. However, it would be great to have the ability to add these images dynamically via the web UI as well
